docs/reference/config.rst

```diff
@@ -427,20 +427,66 @@ the ``color`` option is set to ``yes``. For example, you might have a section
 in your configuration file that looks like this::
 
     ui:
-        color: yes
         colors:
-            text_success: green
-            text_warning: yellow
-            text_error: red
-            text_highlight: red
-            text_highlight_minor: lightgray
-            action_default: turquoise
-            action: blue
+            text_success: ['bold', 'green']
+            text_warning: ['bold', 'yellow']
+            text_error: ['bold', 'red']
+            text_highlight: ['bold', 'red']
+            text_highlight_minor: ['white']
+            action_default: ['bold', 'cyan']
+            action: ['bold', 'cyan']
+            # New colors after UI overhaul
+            text: ['normal']
+            text_faint: ['faint']
+            import_path: ['bold', 'blue']
+            import_path_items: ['bold', 'blue']
+            added:   ['green']
+            removed: ['red']
+            changed: ['yellow']
+            added_highlight:   ['bold', 'green']
+            removed_highlight: ['bold', 'red']
+            changed_highlight: ['bold', 'yellow']
+            text_diff_added:   ['bold', 'red']
+            text_diff_removed: ['bold', 'red']
+            text_diff_changed: ['bold', 'red']
+            action_description: ['white']
 
 Available colors: black, darkred, darkgreen, brown (darkyellow), darkblue,
 purple (darkmagenta), teal (darkcyan), lightgray, darkgray, red, green,
 yellow, blue, fuchsia (magenta), turquoise (cyan), white
 
+Legacy UI colors config directive used strings. If any colors value is still a
+string instead of a list, it will be translated to list automatically. For
+example ``blue`` will become ``['blue']``.
+
+terminal_width
+~~~~~~~~~~~~~~
+
+Controls line wrapping. Defaults to ``80`` characters::
+
+    ui:
+        terminal_width: 80
+
+length_diff_thresh
+~~~~~~~~~~~~~~~~~~
+
+FIXME::
```

Comment:
Looks like it's maximum length in seconds that the importer UI does not consider important. Above this limit, the importer will use text_highlight to note the differences in track runtime between the proposed metadata and the existing track. It does not appear to affect the autotagger matching or difference score calculation in matching at all. https://github.com/beetbox/beets/blob/e10b955a931e4c205b0cadf0860797c0aeee736c/beets/ui/commands.py#L391

Reply:
Can we change the code to >= ? it feels more intuitive.

Reply:
For the docs, something like "Beets compares the length of the imported track with the length stored in the proposed metadata provider (Musicbrainz, Discogs, etc.). If any tracks differ by at least `length_diff_thresh` seconds, they will be colored with `text_highlight`. Below this threshold, different track lengths are colored with text_highlight_minor."

Reply:
Thanks so much for investigating and describing. That is indeed very helpful! Awesome teamwork here! 
I wanted to shorten the paragraph more but failed. It's almost exactly what you suggested:
![Screenshot 2023-10-12 at 18 35 05](https://github.com/beetbox/beets/assets/2733783/6297d88b-ac27-4290-b437-ef99a5884b41)



You agree were I link to in the very last braces (see ....)
Unfortunately `distance_weights` is undocumented. I'm not sure how to properly link to config_default.yaml - or if that should better be fixed up nicely by including that config directive properly in the docs.....

Reply:
Distance_weights is undocumented, I say just mark it in `monospace` and leave the documentation to another day.

Reply:
We would need to change the code from > to >= when comparing length_diff_thresh to match the doc (and what is more intuitive imo). https://github.com/beetbox/beets/blob/3024e6eb425d37e76f0b3ddc93369aea38ad93ad/beets/ui/commands.py#L496

Regarding doc length, yes I am verbose, but I've never wished a doc was shorter as a user. :)

Reply:
@Serene-Arc should we change that as @RollingStar suggested?

Reply:
Both of you FYI @RollingStar @Serene-Arc I fixed that as suggested. It now fits with the docs and I agree it is now more intuitive.

I was curious now @RollingStar, since I was not sure how the word "threshold" is actually defined. This dictionary description implies a "greather than" though:

"the level or point at which you start to experience something, or at which something starts to happen"

https://dictionary.cambridge.org/us/dictionary/english/threshold#google_vignette

Reply:
I don't think there's any harm in changing that. I agree that it makes more intuitive sense if it's `>=` and it's so minor that most people won't even notice. As long as all of the tests pass, it should be fine.